metricbeat/tests/system/test_autodiscover.py

```diff
@@ -25,7 +25,7 @@ def test_docker(self):
                 'docker': {
                     'templates': '''
                       - condition:
-                          equals.docker.container.image: memcached:1.5.3
+                          equals.docker.container.image: memcached:latest
```

Comment:
I'm torn here. In general I prefer that we use fixed versions for consistency but it seem for the autodiscovery test the content does not really matter.